admin-portal/src/pages/packageDetails/components/orderItemsTable/orderItemsTable.tsx

```diff
@@ -34,13 +48,18 @@ export default function OrderItemsTable({
               {item.period.day}/{item.period.month}/{item.period.year}
             </td>
             <td className="actions">
-              <button type="button" onClick={() => onEditItemButtonClick(item)}>
+              <button
+                type="button"
+                onClick={() => onEditItemButtonClick(item)}
+                disabled={disableBtns}
```

Comment:
I don't think we need to maintain a separate state, because the aidpackage is already a prop. We could rather define a variable or just define the logic here.

Reply:
sure, will make the change